src/Common/Core/Impl/Collections/EnumerableExtensions.cs

```diff
@@ -10,12 +10,12 @@ public static class EnumerableExtensions {
         public static List<T> AsList<T>(this IEnumerable<T> source) => source as List<T> ?? source.ToList();
         public static T[] AsArray<T>(this IEnumerable<T> source) => source as T[] ?? source.ToArray();
 
-        public static IEnumerable<T> Append<T>(this IEnumerable<T> source, T item) {
-            foreach (T sourceItem in source) {
-                yield return sourceItem;
-            }
-            yield return item;
-        }
+        //public static IEnumerable<T> Append<T>(this IEnumerable<T> source, T item) {
```

Comment:
It conflicts with another Append apparently available from Linq on 4.6 (but not on net core)